task/build-image-manifest/0.1/build-image-manifest.yaml

```diff
@@ -105,3 +105,26 @@ spec:
       capabilities:
         add:
           - SETFCAP
+
+  - name: push-dockerfile
+    image: quay.io/konflux-ci/oras@sha256:5d0a8a5535fcc4ba467264cacbdeab2fb8662a538a61cb7fc8b3155e3f20fa39
+    env:
+    - name: IMAGE_DIGEST_FILE
+      value: $(results.IMAGE_DIGEST.path)
+    script: |
+      suffix=".dockerfile"
+      image_without_tag=${IMAGE%:*}
+      # remove the potential existing newline
+      manifest_digest=$(cat "${IMAGE_DIGEST_FILE}" | tr -d '\r\n')
+      df_image=${image_without_tag}:${manifest_digest/:/-}${suffix}
+      artype=application/vnd.konflux.dockerfile
```

Comment:
Could you expand `ar` in the variable name?

Reply:
Updated.